examples/async_tcp_stream/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "async_tcp_stream"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+wasmedge_wasi_socket = {path = "../.."}
+tokio = {version = "1.18.1", features=["io-util"]}
```

Comment:
no need

Reply:
Actually I am using `AsyncRead` and `AsyncWrite` traits from `tokio`, but we can implement them as well.